libminifi/src/utils/file/PathUtils.cpp

```diff
@@ -84,13 +86,58 @@ std::string PathUtils::getFullPath(const std::string& path) {
 #endif
 }
 
-std::string PathUtils::globToRegex(std::string glob) {
+std::string globToRegex(std::string glob) {
   utils::StringUtils::replaceAll(glob, ".", "\\.");
   utils::StringUtils::replaceAll(glob, "*", ".*");
   utils::StringUtils::replaceAll(glob, "?", ".");
   return glob;
 }
 
+space_info space(const path p, std::error_code& ec) noexcept {
```

Comment:
Please rename 'p' to something more searchable as it has a bit larger scope. If it clashes with the type name it is usually recommended to use the '_' suffix in these cases.

Reply:
I'm not a fan of trailing underscores unless really necessary, so I went with `path` and let them clash. Let me know if you're fine with this approach.

Reply:
I'm fine with that as well.